ludwig/schema/defaults/defaults.py

```diff
@@ -57,6 +58,11 @@ def get_defaults_jsonschema():
     combiner's field constraints."""
     preproc_schema = schema_utils.unload_jsonschema_from_marshmallow_class(DefaultsConfig)
     props = preproc_schema["properties"]
+
+    # TODO:  Is this the best approach?  Discuss with ksbrar
+    # Special handling for image augmentation, overwrite default with list version
+    props[IMAGE]["properties"]["augmentation"] = get_augmentation_jsonschema()
```

Comment:
I don't think is necessary because you've already added augmentation onto `ImageInputFeatureConfigMixin`

Reply:
commented out, TODO to clean up before merge